### ISSUE TITLE
Default shortcut for "Maximize" doesn't work with mapped modifier key

### ISSUE THREAD

**Issue by ct-clearhaus** [rxhanson/Rectangle#613]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

(At least I was unable to find anything alike when searching for "modifer" and "caps lock".)

macOS version: 11.6
Rectangle version: both v0.48 (53) and v0.49 (54) tested
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): N/A

I've switched around Control and Caps Lock (Settings -> Keyboard -> Keyboard -> Modifier Keys... -> select "^Control" to the right of "Caps Lock (⇪) Key:" and select "⇪ Caps Lock" to the right of "Control (^) Key:").

The keyboard shortcut for "Left Half" works fine when I press ⇪ (which then means Control), alt/⌥ and ← on the physical keyboard; likewise for "Right Half", etc. However, when I press ⇪ (which then means Control) , alt/⌥ and Enter it doesn't fullscreen the window. Though, if I press Control (which then means Caps Lock), alt/⌥ and Enter it does work.

In conclusion, it could seem that the remapped modifier keys are taken into account most places, but not for Maximize, or not in combination with Enter, or ...

(I do notice that both physical keys Control and ⇪ works as Control for Left Half, Right Half, etc.)

**Comment by rxhanson** [comment]
<img width="558" alt="sshot" src="https://user-images.githubusercontent.com/13651296/139882611-878f252a-9044-4eab-aed3-865c7434996e.png">

**Comment by ct-clearhaus** [comment]
👈 🤦‍♂️

Though, unfortunately it didn't make a difference. I tried the otherwise magic trick, stopping and starting Rectangle.app — I'll do a reboot when it suits and test again 🙂

**Comment by rxhanson** [comment]
Doing some housekeeping and going to close this one out since it's been a while and I don't think I'll make any code changes for it. Feel free to comment if there's any further development.
